modules/change-collection-method-ebpf.adoc

```diff
@@ -13,7 +13,7 @@ If the cluster that you are upgrading contains the `SecuredCluster` CR, you must
 . In the {ocp} web console, go to the {product-title-short} Operator page.
 . In the top navigation menu, select *Secured Cluster*.
 . Click the instance name, for example, *stackrox-secured-cluster-services*.
-. Click YAML to open the YAML editor.
-. Locate the `spec.perNode.collector.collection` attribute.
-. If the value is `KernelModule`, then change it to `EBPF`.
-. Click *Save*.
\ No newline at end of file
+. Use one of the following methods:
+* In the *Form view*, under *Per Node Settings* -> *Collector Settings* -> *Collection*, select *CORE_BPF*.
+* Click *YAML* to open the YAML editor and locate the `spec.perNode.collector.collection` attribute. If the value is `KernelModule`, then change it to `EBPF`.
```

Comment:
The point here is that the collection method should not be KernelModule if a customer is upgrading to 4.3. Lines 5, 6, and 9 need to be changed to reflect that. Unfortunately I cannot select those lines to add comment and suggest a changes.

For line 5 it could be changed to

```
[id="change-collection-method-from-kernel-module_{context}"]
```
There may be technical reasons why that change should not be made and is not needed, that you are aware of and I am not.

I suggest changing line 6 to 
```
= Changing the collection method to EBPF or CORE_BPF
```

I suggest changing line 9 to 
```
If the cluster that you are upgrading contains the `SecuredCluster` CR, you must ensure that the per node collection setting is not set to KernelModule. If collection is set to KernelModule change it to EBPF or CORE_BPF. EBPF is the preferred value for collection, unless there is some known reason to use CORE_BPF.
```

You could change lines 17 and 18 to
```suggestion
* In the *Form view*, under *Per Node Settings* -> *Collector Settings* -> *Collection*, select *EBPF* or *CORE_BPF*.
* Click *YAML* to open the YAML editor and locate the `spec.perNode.collector.collection` attribute. If the value is `KernelModule`, then change it to `EBPF` or `CORE_BPF`.
```